client/src/app/interfaces/article.ts

```diff
@@ -0,0 +1,7 @@
+export interface Article {
```

Comment:
Instead of remaking the interface, could you reference the one that the backend uses, so that your frontend is expecting what the backend returns? Specifically, when creating a user, the body that is sent to the backend API should conform to https://github.com/waterseeker/js-rest-api/blob/master/src/articles/dto/create-article.dto.ts and responses from the articles endpoint should be an array of https://github.com/waterseeker/js-rest-api/blob/master/src/articles/article.entity.ts.

Reply:
I'll make this change in another PR, like you suggested on Slack. Thanks!